src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
this can be removed, the psvBrakes and trlBrakes wont cause a non null assertion if you amend lines 60 and 61 to use the ! operator instead of ?, then all instances of this.trlBrakes! and this.psvBrakes! can have the ! removed from lines 206, 211 and 212